### PR TITLE
VB Continuation Comments.

### PR DESCRIPTION
**Pull request by AdamSpeight2008**
@jcouv I've refactored @paul1956 pull-request from #28327.

### CODE REVIEW

File: src/Compilers/VisualBasic/Portable/Errors/Errors.vb [hunk 1]
```diff
@@ -1742,6 +1742,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         ERR_ValueTupleResolutionAmbiguous3 = 37305
 
+        ERR_CommentsAfterLineContinuationNotAvailable1 = 37306
```
Comment (AdamSpeight2008) [inline src/Compilers/VisualBasic/Portable/Errors/Errors.vb]: This error shouldn't be needed now, as it handle by the `Parser.CheckFeatureAvailability` code.

**Comment by CyrusNajmabadi** [comment]
Was this something Paul requested?

**Comment by AdamSpeight2008** [comment]
@CyrusNajmabadi Not exactualy, this was my solution to ... 
> @jcouv **wrote:**
> Can we fold this code into the above, by adding a flag (such as "shouldSkipWhitespace") and making the skipping of whitespace conditional on it?

Just see if would help. I meant no slight against @paul1956 .

**Comment by paul1956** [comment]
@AdamSpeight2008 I like your use of TryGet, if this PR needs any more work I might change to using it. I think what I have now is very clear.

**Pull request closed without merging**